src/EnergyPlus/WaterThermalTanks.hh

```diff
@@ -805,12 +805,12 @@ namespace WaterThermalTanks {
                                       int InNodeNum,
                                       bool FirstHVACIteration,
                                       WaterHeaterSide WaterThermalTankSide,
-                                      int PlantLoopSide,
+                                      DataPlant::LoopSideLocation PlantLoopSide,
                                       bool PlumbedInSeries, // !unused1208
                                       DataBranchAirLoopPlant::ControlTypeEnum BranchControlType,
                                       Real64 OutletTemp,
                                       Real64 DeadBandTemp,
-                                      Real64 SetPointTemp);
+                                      Real64 SetPointTemp_loc);
```

Comment:
Renamed variable to be the same as in the [function definition](https://github.com/NREL/EnergyPlus/blob/770707b069a8b364fb8e91b9604f18e56886f179/src/EnergyPlus/WaterThermalTanks.cc#L10604).